### PR TITLE
update doc for App  and requester secrets

### PR DESCRIPTION
**Pull request by ericr6**
# Before contributing

- [ ] My code has been formatted as explained in [CONTRIBUTING.md](https://github.com/iExecBlockchainComputing/documentation/blob/v8-staging/CONTRIBUTING.md)

### CODE REVIEW

File: for-developers/confidential-computing/app-developer-secret.md [hunk 1]
```diff
@@ -222,8 +256,8 @@ Update the `sconify.sh` script with the variables as follow:
 
 ```bash
 # Declare image related variables
-IMG_NAME=tee-scone-count-api
-IMG_FROM=<docker-hub-user>/count-api:1.0.0
+IMG_NAME=tee-scone-secret-fonction
```
Comment (raorla) [inline for-developers/confidential-computing/app-developer-secret.md]: IMG_NAME=tee-scone-secret-fonction to IMG_NAME=tee-scone-secret-function

File: for-developers/confidential-computing/requester-secrets.md [hunk 2]
```diff
@@ -256,18 +295,25 @@ At this stage, your application is ready to be tested on iExec with the followin
 
 For simplicity, we will use secrets in a TEE-debug app on a debug workerpool. The debug workerpool is connected to a debug Secret Management Service so we will send the requester secrets to this SMS (this is fine for debugging but do not use to store production secrets).
 
+In this example, the app developer's secret must be strictly defined in the following format `a;b;c;d`.
+For example set your secret to `1;1;1;1` for coeficients a,b,c,d.
+For details, go to [Use app secret](app-developer-secret.md####Push-an-application-developer-secret-to-the-SMS)
+
+```bash
+iexec app <appAddress> push-secret
```
Comment (raorla) [inline for-developers/confidential-computing/requester-secrets.md]: remove appAddress
iexec app push-secret

File: for-developers/confidential-computing/app-developer-secret.md (hunk 1 above)
Comment (ericr6) [reply for-developers/confidential-computing/app-developer-secret.md]: done

**Review by raorla — APPROVED** [review]
Test ok

**Pull request merged**